doc/ref_model/chapters/chapter03.md

```diff
@@ -185,4 +185,31 @@ Networking, alongside Compute and Storage, is an integral part of the Cloud Infr
 
+The following diagram represents a high level Network Model detailing, form the network perspective, major subcomponents of the Virtual Resources, Physical Resources, and Management Software components of the NFVI Model Overview in Figure 3-1.The details of this model are discussed in Section 4.3.
 
+<p align="center"><img src="../figures/RM-Chapter 3-Network-Model.png" alt="Network Model Overview" Title="Network Model Overview" width="65%"/></p>
+<p align="center"><b>Figure 3-5:</b> Network Model Overview</p>
+
+Principles that should be followed during the development and definition of the networking scope for the Reference Model, Reference Architectures, Reference Implementations and Reference Conformance test suites:
+
+• Abstraction: A standardized network abstraction layer between the Virtualisation Layers and the Network Physical Resources Layer that hides (or abstracts) the details of the Network Physical resources from the Virtualisation Layers.
+
+> **Note:**  In deployment phases this principle may be applied in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. The network abstraction layer supports, for example, physical resources with or without programmable hardware acceleration, or programmable network switches
+
+•Agnosticity : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+    Control, User and Management traffic types
+    Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads
+    
+•Automation: Enable end-to-end automation, from Physical Fabric installation and provisioning to automation of workload onboarding.
+
+•Openness: All Networking is based on open source or standardized APIs (North Bound Interfaces (NBI) and South Bound Interfaces (SBI)) and should enable integration of open source networking components (e.g. SDN controllers).
+
+•Programmability: Should enable a programmable forwarding plane controlled from a separately deployed control plane.
+
+•Scalability: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+•Workload agnostic: Should be capable to to provide connectivity to any type of workloads, including VNF , CNF and BareMetal workloads.
+
+•Telco Grade: Should support deployments of the telco grade worklaods.
```

Comment:
Nitpick: worklaods-> workloads

Also, I think "telco grade" needs some qualification. Are we talking just the functionality (i.e. confirming to x,y,z standards) our also the non functionality (i.e. low latency, etc).

Reply:
I agree but this is not the place for a long dissertation on the topic.  I agree with your previous comments and changed the principles wording to sound more like  principles.  In the spirit of this, a principle needs to be succinct and that's why we can't define "telco grade" here.  I do not think this is a major problem, as people use this term everyday. Let us find whether it was already defined somewhere and if not to define it somewhere.